src/main/resources/templates/CustomisedException.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html xmlns:th="http://localhost" xmlns:layout="http://www.w3.org/1999/xhtml" layout:decorator="default">
+<head>
+    <title>Exception</title>
+</head>
+<body>
+<div layout:fragment="content">
+    <label>Player One</label>
+    <label>Status: </label>
+    <label>Exception</label>
+    <div>
+        <div class="col"><img class="Centered" src="/images/game_over_404.png" alt="Game Over"/></div>
+        <div class="col"><a href="/">Insert Coin To Continue</a></div>
+        <div class="col"><img src="/images/coin.png" alt="Game Over"/></div>
+    </div>
+</div>
+</body>
+</html>
+
+<!--Copyright 2016-2018 Diffblue Limited. All rights reserved.-->
```

Comment:
instead of having separate pages for errors vs exceptions, if an exception occurs, can it simply load the above error page (as it's more detailed), with an http status code of 500 along with the exception message (if present)?

Reply:
I will do this at a later stage when I figure out how to capture the exception message and display it. For now I think it is helpful to separate it so when an exception occurs it can be easy to tell from Http request errors.